assets/javascript/card.js

```diff
@@ -1,19 +1,89 @@
+const SENSITIVE_CARD = '.sensitive-card';
+const SENSITIVE_CARD_CONTAINER = '.sensitive-card-container';
+
+const updateClasses = f => items => items.forEach(item => f(item.classList));
+const hideAll = updateClasses(classes => classes.add('hidden'));
```

Comment:
Maybe a const?
const HIDDEN = '.hidden'
Other candidates possibly '.open' and '.closed'?

Reply:
I've done this - would generally wait until its used a few times or across different places, rather than just a couple of places (over adjacent lines).

I've kept open and close separate as they are encapsulated in the wrappers and potentially each components has different reasons to change:

> Duplication is far cheaper than the wrong abstraction
https://sandimetz.com/blog/2016/1/20/the-wrong-abstraction